nncf/__init__.py

```diff
@@ -90,3 +90,15 @@
         "Please install one of the supported frameworks above in order to use NNCF on top of it.\n"
         "See the installation guide at https://github.com/openvinotoolkit/nncf#installation-guide for help."
     )
+
+if _AVAILABLE_FRAMEWORKS["torch"]:
+    from nncf import torch as torch
```

Comment:
Do PyTorch extensions compile during this call?

Reply:
No, extensions compiles on first using